client/app/views/menu.coffee

```diff
@@ -12,6 +15,23 @@ module.exports = class MenuView extends ViewCollection
 
     events: ->
         'click .calendars': 'toggleDropdown'
+        'click .calendar-add': 'onAddCalendar'
+
+    onAddCalendar: ->
+        @tag = app.tags.getOrCreateByName "new calendar"
+        # Since an event is needed to create a calendar, let's make a false one
+        calendarEvent = new Event
+                start: moment("19010101", "YYYYMMDD")
+                end: moment("19010101", "YYYYMMDD")
+                description: ''
+                place: ''
+                tags: [t("new calendar")]
+        # saving the event, creating calendar
+        calendarEvent.save {}
+        #a timeout is needed for the created calendar to appear on bottom of the menu items
+        setTimeout ->
+            $('#menuitems li.tagmenuitem:last-of-type .calendar-rename').trigger("click")
+        ,100
```

Comment:
This doesn't work for me, because 100ms is not enough. You could do:
```coffeescript
calendarEvent.save null,
    wait: true
    success: ->
        $('#menuitems li.tagmenuitem:last-of-type .calendar-rename').trigger 'click'
```